js-exercises/sequential-promise/sequentialPromise.js

```diff
@@ -0,0 +1,13 @@
+const sequentialPromise = promises => {
```

Comment:
Are you accepting promises or promise-generating functions? You are executing `promises` which means they are functions, but the name suggests they are an array of promises.

Reply:
I am accepting an array of promises so the name is incorrect. It should be promisesArray. I'll correct it.